utah-design-system-website/src/react/components/websiteContent/Search.jsx

```diff
@@ -3,6 +3,7 @@ import { useEffect, useState } from 'react';
 
 export function Search() {
   const [query, setQuery] = useState('');
```

Comment:
I believe we are trying to `useImmer` everywhere instead of `useState` to help prevent errors with trying to set state outside of a setState function. Of course, useImmer for a string value is trite, but it would be consistent.